specs/regional_communities.md

```diff
@@ -1,61 +1,333 @@
+
+If the developers decide anyways to give an option to create a new community from scratch, then the following should be required actions:
+- Select a geographical area that the community covers, selecting from a list of already existing areas, preferably from a hierarchy of existing areas (global-continent-country-region-municipality-city)
+- Select which one language the community will be using
+- Select which vertical constellation the community will be focused on (multiselect, optional). If no constellation is selected then the geographical area covered should be very small (i.e. country or below)
+- The creator of the community becomes the default Council member by default
+
+## Actor/Action Matrix
+<!-- Describe which action is done by which actors. Feel free to use a table format or provide your own graphics. A "swimlane process chart" often works well here. -->
+REPLACE THIS TEXT
+
+|         | Admin | Member | Non-member Human | Non-member User |
 | ------- | -------- | -------- | -------- | -------- |
-| Actor 1 |          |          |          |          |
-| Actor 2 |          |          |          |          |
-| Actor 3 |          |          |          |          |
-| Actor 4 |          |          |          |          |
+| Find and Follow Community |          |          | Yes         | Yes         |
+| Join a Community |          |          | Yes         |          |
+| Approve a new Community member | Yes         |          |          |          |
+| Split a Community | Yes         |          |          |          |
+| Merge two Communities | Yes         |          |          |          |
+| Interact with my Community | Yes         | Yes         |          |          |
+| Ban a user from a Community | Yes         |          |          |          |
+| Manage a Community | Yes         |          |          |          |
+| Create a new Community |          |          | Yes         |          |
+
+## Data Architecture
+<!-- Describe the data architecture and principles -->
+
+### Geographies
+To prevent unmanageable complexity in the geography dimension, we propose that there should always be a "trunk" that sets the direction into narrower and narrower geographies, from which the language-branches and constellation-branches can expand. The trunk concept is the family of communities that don't have a constellation selected and that has English as their language. Only communities with these characteristics are allowed to create new "child" geographies. Conversely, any Commmunity that either has a non-english language or has narrowed down their Constellation will not be allowed to create new geographies. Instead they will be confined to only select pre-defined geographies from the trunk. 
+
+We should create an initial set of "trunk" Communities
+- Global
+   - North America
+      - Canada
+      - USA
+   - Asia
+      - Indonesia
+      - Japan
+      - South Korea
+      - Russia
+      - Vietnam
+   - South America
```

Comment:
if we have regions, then South America should be also split down into countries.

Reply:
I had a thought that we should just follow some conventional list of countries. But I don't like it. Too much risk that someone will complain about inclusion or exclusion (Gaza, Kurdistan, Cyprus, Taiwan, etc). It also gets hard to manage when you go inside countries. So I prefer an open and evolving hierarchy as proposed.

Reply:
add more then less. I thing there will be more people happy if we add Taiwan then exclude it at all.